crates/pathfinder/src/rpc/api.rs

```diff
@@ -874,42 +874,22 @@ impl RpcApi {
             .and_then(|x| x)
     }
 
-    /// Call a starknet function without creating a StarkNet transaction.
+    /// Call a starknet transaction locally.
     pub async fn call(
         &self,
         request: Call,
         block_hash: BlockHashOrTag,
```

Comment:
Slightly unrelated, but looks like we'll have to follow-up this with updating the JSON-RPC API to use block id: https://github.com/starkware-libs/starknet-specs/blob/v0.1.0/api/starknet_api_openrpc.json#L383

Reply:
Noted. Good thing I already support integers at python side :)

Reply:
Done in a follow-up PR: https://github.com/eqlabs/pathfinder/pull/490